app/Main.hs

```diff
@@ -22,7 +22,13 @@ import Options.Applicative.Simple (addCommand, simpleOptions, simpleVersion)
 import Paths_bench_show (version)
 
 import Options.Applicative
+
+#ifndef NO_CHARTS
 import BenchShow
+#else
+import BenchShow.Internal.Common
+import BenchShow.Internal.Report
+#endif
```

Comment:
It may be cleaner to put the charts CLI functionality in a separate module, and just use addCommand in this module if charts is configured. But we can do that in a separate PR.